fe/.eslintrc.yml

```diff
@@ -5,57 +5,57 @@ extends:
   - airbnb-typescript
   - airbnb/hooks
   - plugin:@typescript-eslint/recommended
-parser: "@typescript-eslint/parser"
+parser: '@typescript-eslint/parser'
```

Comment:
I guess we'll be changing these quotes back and forth until the end of times

(no need to change back)